custom_components/waste_collection_schedule/waste_collection_schedule/source/geoport_nwm_de.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import requests
+import urllib
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+TITLE = "Landkreis Nordwestmecklenburg"
+DESCRIPTION = "Source for Landkreis Nordwestmecklenburg"
+URL = "https://www.geoport-nwm.de/de/abfuhrtermine-geoportal.html"
+TEST_CASES = {
+    "Rüting": {"district": "Rüting"},
+    "Grevenstein u. ...": {"district": "Grevenstein u. Ausbau"},
+    "Seefeld": {"district": "Seefeld/ Testorf- Steinfort"},
+    "1100l": {"district": "Groß Stieten (1.100 l Behälter)"},
+    "kl. Bünsdorf": {"district": "Klein Bünsdorf"}
+}
+
+
+class Source:
+    def __init__(self, district):
+        self._district = district
+        self._ics = ICS()
+
+    def fetch(self):
+        arg = convert_to_arg(self._district)
+        today = datetime.date.today()
+        year = today.year
+        r = requests.get(
```

Comment:
Other sources that have the same year issue implement the following behavirour: In December they try to get the data for the current year and in addition the data for the following year (see `fetch_year` methods). Could please implement the same behaviour?